FIAT/guzman_neilan.py

```diff
@@ -30,29 +43,68 @@ def ExtendedGuzmanNeilanSpace(ref_el, subdegree, reduced=False):
         B = modified_bubble_subspace(B)
 
     if reduced:
-        BR = BernardiRaugel(ref_el, sd, subdegree=subdegree).get_nodal_basis()
+        BR = BernardiRaugel(ref_el, order).get_nodal_basis()
         reduced_dim = BR.get_num_members() - (sd-1) * (sd+1)
         BR = BR.take(list(range(reduced_dim)))
     else:
-        BR = ExtendedBernardiRaugelSpace(ref_el, subdegree)
+        BR = ExtendedBernardiRaugelSpace(ref_el, order)
+
     GN = constant_div_projection(BR, C0, B)
+    if kind == 2:
+        Bk = take_interior_bubbles(C0, order)
+        GN = polynomial_set.polynomial_set_union_normalized(GN, Bk)
     return GN
 
 
 class GuzmanNeilan(finite_element.CiarletElement):
```

Comment:
Why is the element `GuzmanNeilan` but you use `Extended` in the space name?

Reply:
Bell is just called bell, but we just have a P5 Argyris with different edge dofs. Should I just name them GuzmanNeilanSpace and GuzmanNeilan?

Reply:
Now nothing is `Extended`, so this is consistent